src/NewChain/Initialization.ts

```diff
@@ -172,6 +174,19 @@ export default class Initialization {
       ),
     ]);
 
+    // Resets ostGateway organization admin address to 0x in origin chain as deployer is admin here.
+    // Resets coGatewayAndOstPrime organization admin to 0x address in aux chain as deployer is admin here.
+    await Promise.all([
+      originChain.resetOrganizationAdmin(
```

Comment:
This should not be part of `createAuxiliaryChain`. It is not directly part of creating an auxiliary chain.
Instead, add a new function and call it after `await Initialization.createAuxiliaryChain()`. Then, you also don't need to pass `initConfig` into `createAuxiliaryChain`.

You will be able to access the auxiliary deployer from the `MosaicConfig.auxiliaryOriginalDeployer`. So you probably need to pass the two deployers to the new function with the data available in the caller.

Reply:
Done!